src/librustdoc/html/render/cache.rs

```diff
@@ -623,13 +603,35 @@ fn build_index(krate: &clean::Crate, cache: &mut Cache) -> String {
         .map(|module| shorten(plain_summary_line(module.doc_value())))
         .unwrap_or(String::new());
 
+    let crate_aliases = aliases
+        .iter()
+        .map(|(k, values)| {
+            (
+                k.clone(),
+                values
+                    .iter()
+                    .filter_map(|v| {
+                        let x = &crate_items[*v];
+                        if x.parent_idx.is_some() == x.parent.is_some() { Some(*v) } else { None }
+                    })
+                    .collect::<Vec<_>>(),
+            )
+        })
+        .filter(|(_, values)| !values.is_empty())
+        .collect::<Vec<_>>();
```

Comment:
I added it because some items were still orphan (they have a parent index but the parent cannot be found). I'll test it again to be sure it's not needed anymore.

Reply:
It's not needed anymore, removing!